lib/cli.js

```diff
@@ -9,14 +9,12 @@
  */
 
 const argv = require('minimist')(process.argv.slice(2))
-const log = require('./utils/log')
 const outputFile = argv.outputFile || argv.o
 const inputFile = argv._[0]
 const secret = argv.secret || argv.s
 const encryptionAlgo = argv.algo || argv.a
 
 const cryptography = require('./cryptography')
 
-
-if (argv.decrypt || argv.d) log(cryptography.decrypt({secret, outputFile, encryptionAlgo}),'info')
-	else cryptography.encrypt({ secret, inputFile, outputFile, encryptionAlgo });
+if (argv.decrypt || argv.d) cryptography.decrypt({ secret, inputFile, outputFile, encryptionAlgo })
```

Comment:
Let the log be here, please do not remove it from here since we need the log to be happening only in case of cli. Not in the decrypt

Reply:
All the logging for decrypt should happen in this file and not in `lib/cryptography.js` unless its an error.